packages/bpk-docs/src/layouts/DefaultLayout/default-layout.scss

```diff
@@ -26,7 +26,8 @@
   }
 
   &__main {
-    flex: 1;
+    display: flex;
+    flex: 1 0 auto;
```

Comment:
This can just be `flex: 1;`

Reply:
That's not true see the [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/flex). Namely

> The item is given the specified proportion of the free space in the container. This is equivalent to setting "flex: <positive-number> 1 0".